loleaflet/Makefile.am

```diff
@@ -353,9 +359,18 @@ LOLEAFLET_JS_WEBORDER =\
 	src/map/anim/Map.PanAnimation.js \
 	src/dom/PosAnimation.Timer.js \
 	src/control/Control.Scroll.Annotation.js \
-	src/layer/marker/DivOverlay.js \
+	src/layer/marker/DivOverlay.js
+
+if ENABLE_FEEDBACK
+LOLEAFLET_JS_WEBORDER +=\
```

Comment:
Adjusted loleaflet/Makefile.am while porting 

- do not use  `LOLEAFLET_JS_LST` from 6-4 commit https://github.com/CollaboraOnline/online/pull/2334/commits/b729a36dcff3be7241076ec0d18bd6db0f9d17eb and instead keep using `LOLEAFLET_JS_WEBORDER`
- Also resolve conflicts  (specially under the line 372 so to not overriding master) etc